tests/test.sh

```diff
@@ -31,12 +31,28 @@ test_bin() {
   local expect=$1; shift
   local RESULT=-1
   cd $name;
-  bundle exec metadata-json-lint $* metadata.json >/dev/null 2>&1
+  bundle exec metadata-json-lint $* metadata.json >last_output 2>&1
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
-    fail "Failing Test '${name}' (bin)"
+    fail "Failing Test (unexpected exit code) '${name}' (bin)"
```

Comment:
Can we point the user to the location of the `last_output` file to review for the failure?